Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -97,8 +97,12 @@ static const char* toString(MemoryUsagePolicy policy)
 
 static size_t thresholdForMemoryKillOfActiveProcess(unsigned tabCount)
 {
+#if CPU(X86_64) || CPU(ARM64)
```

Comment:
```suggestion
#if WTF_CPU_ADDRESS64
```

Reply:
Actually, I guess this would work:

```suggestion
#if CPU(ADDRESS64)
```